test/test_common.py

```diff
@@ -49,6 +50,20 @@ def get_pgpass_contents():
         os.environ["HOME"] = original_home
 
     def test_pg_versions(self):
```

Comment:
I would split this in smaller tests and move it outside of `TestCommon`, it seems we're not using `self` at all.

Reply:
Done